kong/router/transform.lua

```diff
@@ -258,6 +259,41 @@ local function gen_for_nets(ip_field, port_field, vals)
 end
 
 
+local function sni_op_transform(sni)
```

Comment:
Could we move `sni_op_transform` and `sni_val_transform` after `is_stream_route`, i.e. the original `sni_val_transform`'s place?
This will be more friendly for reviewers.

Reply:
ok